usaspending_api/config/tests/unit/test_config.py

```diff
@@ -1079,8 +1079,8 @@ def test_override_with_dotenv_file(tmpdir):
     shutil.copy(str(_PROJECT_ROOT_DIR / ".env.template"), dotenv_file)
     if Path(_PROJECT_ROOT_DIR / ".env").exists():
         shutil.copy(str(_PROJECT_ROOT_DIR / ".env"), dotenv_file)
-    with open(dotenv_file, "a"):
-        dotenv_file.write(f"COMPONENT_NAME={dotenv_val}", "a")
+    with open(dotenv_file, "w"):
```

Comment:
Don't see a reason to change this from append to write, but also won't block over it since it seems to be functioning the same for the purpose of the test.

Reply:
I changed it for the sake of consistency and for since it doesn't change the way the test works, rather whether us writing to the file as opposed to appending. I think we should keep this?